assets/js/base/components/cart-checkout/index.js

```diff
@@ -0,0 +1,7 @@
+export { default as ProductImage } from './product-image';
+export { default as ProductLowStockBadge } from './product-low-stock-badge';
+export { default as ProductMetadata } from './product-metadata';
+export { default as ProductName } from './product-name';
+export { default as ProductPrice } from './product-price';
+export { default as ProductSaleBadge } from './product-sale-badge';
+export { default as ProductVariationData } from './product-variation-data';
```

Comment:
I noticed some of these blocks are very similar to the ones in `atomic/components/product/`. I think it might be worth checking if they can be merged, but I left it out of this PR because it was already quite big and I don't think that's something blocking.

Reply:
Agree, this is good follow up for later. Great that cart & checkout are sharing these components now though :)